### PR TITLE
adjust CCS in ar6

### PR DESCRIPTION
**Pull request by orichters**
This PR replaces #19 which wrongly associated CCS with `Carbon Management|Carbon Capture`, not `Carbon Management|Storage`. To enable comparison, I preferred a new PR instead of changing the old one.
- I ran some scripts that identifier things already specified in NAVIGATE, but not in AR6, and added them 
- the NGFS people criticized that Carbon Sequestration|CCS|Biomass|... variables evolve completely in the negative territory, which is probably a sign problem. I think it should be fixed with this

### CODE REVIEW

File: ar6/mapping_template_AR6.csv [hunk 1]
```diff
@@ -24,28 +24,29 @@ idx;Variable_AR6;Unit;r21m42;r21m42_unit;r21m42_factor;r21m42_spatial;m42;m42_un
...
-;Carbon Sequestration|CCS|Fossil;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Pe2Se;Mt CO2/yr;;;;;;;;;;;;;;;
+24;Carbon Sequestration|CCS|Fossil;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Energy|Demand|Industry;Mt CO2/yr;;;;;;Carbon Management|Storage|+|Fossil|Pe2Se;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from fossil fuel use and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
+;Carbon Sequestration|CCS|Fossil;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Pe2Se;Mt CO2/yr;;;;;;Carbon Management|Storage|Industry Energy|+|Fossil;Mt CO2/yr;;;;;;;;
+;Carbon Sequestration|CCS|Fossil|Demand;Mt CO2/yr;;;;;;;;Carbon Management|Carbon Capture|Industry Energy|+|Fossil;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
```
Comment (strefler) [inline ar6/mapping_template_AR6.csv:41]: Looks like this is a new variable - then it should be #25 right?
Plus it should alse be Carbon Management|Storage|...

File: ar6/mapping_template_AR6.csv [hunk 2]
```diff
@@ -24,28 +24,29 @@ idx;Variable_AR6;Unit;r21m42;r21m42_unit;r21m42_factor;r21m42_spatial;m42;m42_un
...
-26;Carbon Sequestration|CCS|Fossil|Energy|Demand|Industry;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Energy|Demand|Industry;Mt CO2/yr;;;;;;;;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in industry (IPCC category 1A2) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
+26;Carbon Sequestration|CCS|Fossil|Energy|Demand|Industry;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Energy|Demand|Industry;Mt CO2/yr;;;;;;Carbon Management|Carbon Capture|Industry Energy|+|Fossil;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in industry (IPCC category 1A2) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
```
Comment (strefler) [inline ar6/mapping_template_AR6.csv:43]: Also Carbon Management|Storage|...

File: ar6/mapping_template_AR6.csv [hunk 3]
```diff
@@ -24,28 +24,29 @@ idx;Variable_AR6;Unit;r21m42;r21m42_unit;r21m42_factor;r21m42_spatial;m42;m42_un
...
-31;Carbon Sequestration|CCS|Fossil|Energy|Supply|Liquids;Mt CO2/yr;;;;;;;;;;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in liquid fuel production (part of IPCC category 1A) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
+28;Carbon Sequestration|CCS|Fossil|Energy|Supply|Electricity;Mt CO2/yr;;;;;;;;Carbon Management|Carbon Capture|Fossil|Pe2Se|+|Electricity w/ couple prod;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in electricity production (part of IPCC category 1A1a) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
```
Comment (strefler) [inline ar6/mapping_template_AR6.csv:46]: same for all of these Carbon Management|Storage|...

File: ar6/mapping_template_AR6.csv (hunk 1 above)
Comment (orichters) [reply ar6/mapping_template_AR6.csv:41]: I have never touched these numbers. Is there an efficient way to increment them all, if that was desired?

File: ar6/mapping_template_AR6.csv (hunk 2 above)
Comment (orichters) [reply ar6/mapping_template_AR6.csv]: fixed

File: ar6/mapping_template_AR6.csv (hunk 3 above)
Comment (orichters) [reply ar6/mapping_template_AR6.csv]: fixed

File: ar6/mapping_template_AR6.csv (hunk 1 above)
Comment (orichters) [reply ar6/mapping_template_AR6.csv]: I have taken it out, as I can't even remember why I put it in.

**Review by strefler — APPROVED** [review]

File: ar6/mapping_template_AR6.csv [hunk 4]
```diff
@@ -24,28 +24,28 @@ idx;Variable_AR6;Unit;r21m42;r21m42_unit;r21m42_factor;r21m42_spatial;m42;m42_un
...
-;Carbon Sequestration|CCS|Biomass;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Biomass|Energy|Demand|Industry;Mt CO2/yr;;;;;;;;;;;;;;;
+14;Carbon Sequestration|CCS|Biomass;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Biomass|Supply|w/ couple prod;Mt CO2/yr;;;;;;Carbon Management|Storage|+|Biomass|Pe2Se;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from bioenergy use and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
+;Carbon Sequestration|CCS|Biomass;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Biomass|Energy|Demand|Industry;Mt CO2/yr;;;;;;Carbon Management|Storage|Industry Energy|+|Biomass;Mt CO2/yr;;;;;;;;
 15;Carbon Sequestration|CCS|Biomass|Energy;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Biomass|Supply|w/ couple prod;Mt CO2/yr;;;;;;Emi|CO2|CDR|BECCS;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from bioenergy use and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
```
[the remark this answers is not in the document]
Comment (orichters) [reply ar6/mapping_template_AR6.csv]: all right, fixed that

**Pull request merged**